other-guides/hosting-on-linux-server

```diff
@@ -0,0 +1,53 @@
+# Hosting on a Linux Server
+
+The best way to host your bot is on a Linux Server (or simply said: just a virtual server. Its also for bare metal servers).
+
+## Requirements
+
+- A Linux Server with a stable version, node, npm, git, pm2. 
```

Comment:
Stable version?? What does that exactly mean?? You can recommend **Debian 10**, as it's a very stable, robust, and minimal Linux distro, best suited for servers.